### PR TITLE
Doc notes for core team members

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

File: doc/source/development/maintaining.rst [hunk 1]
```diff
@@ -189,5 +196,29 @@ being helpful on the issue tracker.
...
+2. You should not merge pull requests that have an active discussion, or pull
+   requests that has any ``-1`` votes from a core maintainer. Pandas operates
+   by consensus.
+3. For larger changes, it's good to have a +1 from at least two core team members.
+
+In addition to the items listed in :ref:`maintaining.closing`, you should verify
+that the pull request is assigned the correct milestone.
+
+Pull requests merged with a patch-release milestone will typically be backported
+by our bot. Verify that the bot noticed the merge (it will leave a comment within
```
Comment (jreback) [inline doc/source/development/maintaining.rst:218]: add ```@Meeseeksdev backport 1.1.x``` to trigger the bot (if needed)

**Comment by jreback** [comment]
> Something went wrong ... Please have a look at my logs.
> 
> It seem that the branch you are trying to backport to does not exists.

lol, trigger the bot with my comment :->

File: doc/source/development/maintaining.rst [hunk 2]
```diff
@@ -189,5 +196,29 @@ being helpful on the issue tracker.
...
+a minute typically). If a manual backport is needed please do that, and remove
```
Comment (simonjayhawkins) [inline doc/source/development/maintaining.rst:219]: in the first instance, could we ask the original PR author to do that

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @TomAugspurger

**Pull request merged**